inc/ui/class-payment-methods-element.php

```
@@ -190,10 +188,10 @@ public function defaults() {
 	 *
 	 * @param array       $atts Parameters of the block/shortcode.
 	 * @param string|null $content The content inside the shortcode.
-	 * @return string
+	 * @return void
 	 */
-	public function output($atts, $content = null) {
+	public function output($atts, $content = null): void {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$atts'. (Unused Code Rules)

(UnusedFormalParameter)